pkg/rpm/resolver/resolver.go

```diff
@@ -158,6 +158,14 @@ func (r *Resolver) writeDockerfile(localPackagesPath string, packages *image.Pac
 	return nil
 }
 
+func (r *Resolver) generateAddRepoList(repos []image.AddRepo) (repoList []string) {
```

Comment:
Does it make sense to return the string which is being used instead?

Reply:
It makes sense, yes. In the `generateAddRepoStr` we will concatenate a string that will be passed to the resolver Dockerfile.